src/tpm2/Marshal.c

```diff
@@ -2203,7 +2203,7 @@ UINT16
 TPM2B_NV_PUBLIC_Marshal(TPM2B_NV_PUBLIC *source, BYTE **buffer, INT32 *size)
 {
     UINT16 written = 0;
-    BYTE *sizePtr;
+    BYTE *sizePtr = NULL;
```

Comment:
How did you find this?

sizePtr is initialized when buffer != NULL and then later on access also again when buffer != NULL.

Reply:
Oh, gcc on s390x.. uuhhh

Reply:
yeah both are gcc on s390x findings.

Reply:
I had these types of findings before also... quite a few of them on different architectures and most (if not all) of them were false positives.

Reply:
thanks for the explanation, I've already updated the branch an hour ago, should be good now.